pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotInstanceRestletResource.java

```diff
@@ -0,0 +1,204 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.controller.api.resources;
+
+import java.util.List;
+import org.apache.helix.model.InstanceConfig;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.linkedin.pinot.controller.api.pojos.Instance;
+import com.linkedin.pinot.controller.helix.core.PinotHelixResourceManager;
+import com.linkedin.pinot.controller.helix.core.PinotResourceManagerResponse;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+
+@Api(tags = Constants.INSTANCE_TAG)
+@Path("/")
+public class PinotInstanceRestletResource {
+  private static final Logger LOGGER = LoggerFactory.getLogger(
+      PinotInstanceRestletResource.class);
+  @Inject
+  PinotHelixResourceManager pinotHelixResourceManager;
+
+  public static  class Instances
+  {
+    List<String> instances;
+    public Instances(@JsonProperty("instances") List<String> instances) {
+      this.instances = instances;
+    }
+
+    public List<String> getInstances() {
+      return instances;
+    }
+
+    public Instances setInstances(List<String> instances) {
+      this.instances = instances;
+      return this;
+    }
+  }
+
+  @GET
+  @Path("/instances")
```

Comment:
In the jersey API the trailing slash does not make a difference. The API will work for both.